landoapi/uplift.py

```diff
@@ -252,45 +299,44 @@ def create_uplift_bug_update_payload(
     return payload
 
 
-UPDATE_RETRIES = 3
-
-
 def update_bugs_for_uplift(
-    bmo_url: str,
-    bmo_api_key: str,
-    changeset_titles: list[str],
     repo_name: str,
-    milestone: int,
+    hgrepo: HgRepo,
+    changeset_titles: list[str],
 ):
     """Update Bugzilla bugs for uplift."""
```

Comment:
More cleanly, I think we could pass the milestone file directly (or a string) from where we are calling this method.

It's not clear that this method needs to be run from within the `hgrepo.for_push` or `hgrepo.for_pull` context managers.